package.json

```diff
@@ -31,7 +31,23 @@
     "onLanguage:dune-workspace",
     "onCommand:ocaml.select-sandbox",
     "onCommand:ocaml.open-terminal",
-    "onCommand:ocaml.open-terminal-select"
+    "onCommand:ocaml.open-terminal-select",
+    "workspaceContains:dune-project",
+    "workspaceContains:**/dune-project",
+    "workspaceContains:*.opam",
+    "workspaceContains:**/*.opam",
+    "workspaceContains:*.ml",
+    "workspaceContains:**/*.ml",
```

Comment:
Are you sure we need both `*.foo` and `**/*.foo`?

From the docs:

> ** to match any number of path segments, including none

So I think `**/.foo` should be enough.

Reply:
Seems to work as well.